packages/gatsby-telemetry/src/telemetry.ts

```diff
@@ -118,6 +120,7 @@ export class AnalyticsTracker {
   repositoryId?: IRepositoryId
   features = new Set<string>()
   machineId: string
+  siteHash?: string = createContentDigest(process.cwd())
```

Comment:
This will give misleading information in electron, and not all events will be sent with a siteHash to override it

Reply:
Updated so that this won't be the default (but memorizing it here for cli to use)